packages/web/src/utils/config.ts

```diff
@@ -47,7 +47,7 @@ export function getBaseWebpackPartial(
           exclude: /node_modules/,
           options: {
             ...createBabelConfig(dirname(options.main), esm, options.verbose),
-            cacheDirectory: true,
+            cacheDirectory: false,
```

Comment:
@phuson did you notice any performance decrease?

Reply:
In my limited testing, the difference does not seem significant. However, I would guess that this will vary quite a bit, depending on the size of the application.